### PR TITLE
Add DAP to pages

### LINKED ISSUE #682: Implement DAP
Blocked by #497

### PR DESCRIPTION
**Pull request by greg-does-weather**
## What does this PR do? 🛠️

Adds the DAP script to all of our pages.

Question for @shadkeene! The [DAP implementation guide](https://github.com/digital-analytics-program/gov-wide-code/blob/master/documentation/GSA%20DAP%204.1%20-%20DAP%20Code%20Capabilities%20Summary%20and%20Reference.pdf) lists some configurations we must and can set. I only set the two listed below, but if you want to add more (or change the ones I've put in), it's easy to change now and in the future.

* agency: `NOAA`
* subagency: `NWS`
* sitetopic: `weather,safey`

Closes #682

### CODE REVIEW

File: tests/a11y/cypress/support/e2e.js [hunk 1]
```diff
@@ -19,3 +19,7 @@ import "./commands";
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+// Ignore uncaught script errors. We can't control what happens with external
+// scripts, so...
+Cypress.on("uncaught:exception", () => false);
```
Comment (greg-does-weather) [inline tests/a11y/cypress/support/e2e.js:25]: This makes me sad, but for now, we get an unhandled script error from DAP which causes the Cypress tests to fail.

File: tests/load-times/cypress/support/e2e.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Ignore uncaught script errors. We can't control what happens with external
+// scripts, so...
+Cypress.on("uncaught:exception", () => false);
```
Comment (eric-gade) [inline tests/load-times/cypress/support/e2e.js:3]: Barely related: we should put in a test to ensure there is no logging to the browser console

**Review by eric-gade — APPROVED** [review]
lgtm 👍

File: tests/load-times/cypress/support/e2e.js (hunk 2 above)
Comment (greg-does-weather) [reply tests/load-times/cypress/support/e2e.js:3]: For our own code, we could have eslint throw an error for console statements (it currently throws warning, but I don't think we prevent deployments over warnings.)

For 3rd-party code (I guess USWDS and DAP?), should we also try to enforce a no-console-logging policy? I suppose it's worth doing now, and if we ever start getting log messages from some 3rd-party place, it'll force us to see it and decide what to do with it.

Comment (eric-gade) [reply tests/load-times/cypress/support/e2e.js:3]: Good idea. Let's stick with eslint for now, that should catch us. But if we start bringing in frameworks with dependencies (like, say, a Vue application with all the fixins), we might want the e2e version

**Pull request merged**